public/static/locales/en/me.json

```diff
@@ -12,16 +12,16 @@
   "close": "Close",
   "support": "Support",
   "addTarget": "Add Target",
-  "noOfTrees": "Number of trees",
+  "noOfTrees": "How many trees?",
   "datePlanted": "Date Planted",
   "speciesIsRequired": "Species is required",
   "treeSpecies": "Tree Species",
-  "drawPolygon": "Please draw a polygon to locate your trees",
+  "drawPolygon": "Please draw the area where you planted the trees. Click on the first location again when you are done",
```

Comment:
I cannot yet see this resource as multiple tree registration does not yet work for me. Is that correct that multiple tree registration does not yet work?

Reply:
If you enter more than 25 trees it will ask you to draw a polygon

Reply:
It never came to my mind to test with higher numbers ;-)